manifests/init.pp

```diff
@@ -243,6 +250,8 @@
   Stdlib::CreateResources                             $manage_dropins = {},
   Stdlib::CreateResources                             $udev_rules = {},
   Boolean                                             $manage_coredump = false,
+  Boolean                                             $manage_nspawn = false,
+  Optional[Enum['systemd-container']]                 $nspawn_package = undef,
```

Comment:
Does it matter which string this is? If you go this route, why not a boolean?

Reply:
This is consistent with the systemd-resolved package Enum.

Presumably the thought that random OS in the future would have a new string and it needs recording somewhere.